courses/migrations/0016_enrollment_order.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 2.1.7 on 2019-06-25 18:21
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def backpopulate_order(apps, schema_editor):
+    """
+    Set the order for existing enrollments.
+    """
+    CourseRunEnrollment = apps.get_model("courses", "CourseRunEnrollment")
+    ProgramEnrollment = apps.get_model("courses", "ProgramEnrollment")
+    Order = apps.get_model("ecommerce", "Order")
+    Product = apps.get_model("ecommerce", "Product")
+
+    def get_order(user, product):
```

Comment:
How confident are we that this will never return `None` for a given enrollment in all our environments? Mainly asking because this PR also sets `NOT NULL` on the relation and that could cause migrations to fail if this assumption doesn't hold.

Reply:
Made order nullable, and logging an error if no matching order is found when backpopulating